polaris-react/src/components/TopBar/components/Menu/Menu.tsx

```diff
@@ -25,6 +28,8 @@ export interface MenuProps {
   onClose(): void;
   /** A string that provides the accessibility labeling */
   accessibilityLabel?: string;
+  /** A boolean property indicating whether a custom activator is being used */
+  hasCustomActivator?: boolean;
```

Comment:
What if this was `customActivatorWidth` so we could remove the magic number in `USER_MENU_POPOVER_WIDTH`?

Reply:
I think that makes more sense than a fixed width, especially since the design was updated to have a `300px` instead. We could also make it more general to `customWidth` since the internal activator is now default to all stores.